pkg/logcli/client/client.go

```diff
@@ -131,14 +131,16 @@ func (c *DefaultClient) Series(matchers []string, from, through time.Time, quiet
 }
 
 // LiveTailQueryConn uses /api/prom/tail to set up a websocket connection and returns it
-func (c *DefaultClient) LiveTailQueryConn(queryStr string, delayFor int, limit int, from int64, quiet bool) (*websocket.Conn, error) {
-	qsb := util.NewQueryStringBuilder()
-	qsb.SetString("query", queryStr)
-	qsb.SetInt("delay_for", int64(delayFor))
-	qsb.SetInt("limit", int64(limit))
-	qsb.SetInt("from", from)
+func (c *DefaultClient) LiveTailQueryConn(queryStr string, delayFor time.Duration, limit int, start time.Time, quiet bool) (*websocket.Conn, error) {
+	params := util.NewQueryStringBuilder()
+	params.SetString("query", queryStr)
+	if delayFor != 0 {
+		params.SetInt("delayFor", int64(delayFor.Seconds()))
```

Comment:
Should be set to `delay_for`?
```suggestion
		params.SetInt("delay_for", int64(delayFor.Seconds()))
```

Reply:
Oh whoops yeah. Wonder how I managed that.